src/main/java/com/smartsheet/api/WorkspaceResources.java

```diff
@@ -46,7 +47,7 @@
 	 * @throws ServiceUnavailableException if the REST API service is not available (possibly due to rate limiting)
 	 * @throws SmartsheetException if there is any other error during the operation
 	 */
-	public List<Workspace> listWorkspaces() throws SmartsheetException;
+	public DataWrapper<Workspace> listWorkspaces(boolean includeAll, Integer pageSize, Integer page) throws SmartsheetException;
```

Comment:
Should we have the users create the PaginationParameters class and pass it to the listWorkspaces() function? Or should they pass the three parameters and we build the PaginationParameters object off of that? If we decide to have the user create the PaginationParameter and pass it to the API endpoints/functions, we'll have to change it for sheets also, and other future resources we work on.